WriterDesk1/src/pages/Settings.js

```diff
@@ -97,6 +97,19 @@ const Settings = () => {
         navigate("/", { replace: true });
     }
 
+	/**
+	 *	Function that makes the api call to delete the user
+	 */
+    const deleteUser = () => {
+        //   The backend url:
+        const url = 'https://127.0.0.1:5000/usersapi/deleteUserSelf';
+        // Make the backend call and set the table data from the response data:
+        axios.post(url,{},{headers: authHeader()}).then((response) => {
+        })
+        window.location.reload();
+        return false;
```

Comment:
waarom return false?

Reply:
Stond ergens op internet dat dat moest na een reload in een onclick maar idk waarvoor. Kan het wel weghalen

Reply:
In principe zou ik er dan return van maken, zonder false. Maar vgm is het helemaal niet nodig, maar dat zou je effe moeten testen